cachi2/core/package_managers/gomod.py

```diff
@@ -1247,6 +1257,13 @@ def __init__(self, repo: git.Repo, commit: git.objects.commit.Commit):
         self._commit = commit
 
     @classmethod
+    @backoff.on_exception(
+        fast_constant_delay,
+        Exception,
+        jitter=None,  # use deterministic backoff, do not apply jitter
+        max_tries=5,
```

Comment:
This number is repeated multiple times across the patch, so I think it deserves a generic constant.